AnnoyancesFilter/sections/popups.txt

```diff
@@ -99,6 +99,8 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+mangaplus.shueisha.co.jp#$#div[class^="index-module_container_"] { display: none !important;}
+mangaplus.shueisha.co.jp#$#body[style="position: fixed;"] { position: static !important; }
```

Comment:
Unfortunately, I don't see the popup.

Reply:
I used Boston IP - though the site is co.jp inaccesible from Japane.

Reply:
I'll leave for a while, please freely close PR and fix on your end if appropriate.

Reply:
It's finally reproduced with US IP. The rules seem to be normal.